tests/test_msi_7zw0led.py

```diff
@@ -0,0 +1,49 @@
+from struct import pack
+
+import pytest
+from _testutils import MockHidapiDevice
+
+from liquidctl.driver.msi_7zw0led import MSI7ZW0LED
+
+
+@pytest.fixture
+def mpgCoreLiquidK360Device():
+    description = 'Mock MPG Series CORELIQUID K360'
+    device = _MockCoreLiquidK360Device(0xffff, 0xb130)
+    dev = MSI7ZW0LED(device, description)
+    return dev
+
+
+class _MockCoreLiquidK360Device(MockHidapiDevice):
+    def read(self, length):
+        buf = bytearray([0xd0, 0x31])
+        buf += pack('h', 496)
+        buf += pack('h', 517)
+        buf += pack('h', 509)
+        buf += pack('h', 1045)
+        buf += pack('h', 1754)
+        buf += bytearray([0, 0, 0, 0, 0x7d, 0, 0x7d, 0, 0, 0])
+        buf += pack('h', 20)
+        buf += pack('h', 20)
+        buf += pack('h', 20)
+        buf += pack('h', 20)
+        buf += pack('h', 50)
+        buf += bytearray([0] * 32)
```

Comment:
Aren't these `pack()` calls fragile regarding different endianess?

Reply:
I think so. They follow USB's endianness so little-endian. Will add the endianness symbols.